src/docdb/nodes.ts

```diff
@@ -133,6 +133,8 @@ export class DocDBDocumentNode implements IDocumentNode {
 
 	readonly contextValue: string = "cosmosDBDocument";
 
+	readonly partitionKeyValue = this.getPartitionKeyValue();
```

Comment:
Made it an accessible readonly variable because I'd ideally only want to compute the partitionKeyValue once, but access multiple times.